docker-compose.yml

```
@@ -11,10 +11,30 @@ services:
       - 4000:4000
     volumes:
       - ./frontend:/usr/src/app
-    command: sh -c 'yarn install && yarn start'
+    command: sh -c 'yarn install && yarn add @nuxtjs/axios && yarn start'
```

あんまりjs周りに詳しくないんやけど、これってyarn installの対象になる依存関係リストに`@nuxtjs/axios`を付け加えておくだけじゃあかんの？`yarn add @nuxtjs/axios`を別途足してるのが気になる👀

---

@hoka-isdl ごめん、依存関係リストはpackage.jsonのことを指しててん。
以下のcommitで`@nuxtjs/axios`は既にpackage.jsonに追記されてるみたいやから、下記のように修正するだけで良さそうかな？動作に問題ないか確認してみてほしい。
```suggestion
    command: sh -c 'yarn install && yarn start'
```

Commit URL: https://github.com/hoka-isdl/MERS/pull/15/commits/da2a89db76c55044e3f1428f7c2971bd6d85f005#diff-da6498268e99511d9ba0df3c13e439d10556a812881c9d03955b2ef7c6c1c655R6

---

サーバーサイドのPRありがとう。この件はどうなった？

---

上記の変更でdockerを起動したところエラーは生じなかったためyarn add の部分は削除いたします。

---

`yarn install`はpackage.jsonに記載されたパッケージを一括インストールするもので、`yarn add`は引数のパッケージをインストールするものみたいやから、ackage.jsonに記載しておけば`yarn install`があるだけで十分そうやね。